boards/alientek-pandora/Makefile.features

```diff
@@ -0,0 +1,11 @@
+CPU = stm32
+CPU_MODEL = stm32l475ve
+
+# Put defined MCU peripherals here (in alphabetical order)
+FEATURES_PROVIDED += periph_rtc
+FEATURES_PROVIDED += periph_rtt
+FEATURES_PROVIDED += periph_timer
+FEATURES_PROVIDED += periph_uart
+
+# Put other features for this board (in alphabetical order)
+FEATURES_PROVIDED += riotboot
```

Comment:
```suggestion
```

Reply:
You can squash those change in right away

Reply:
Or never mind the patch I can add it in a follow-up, just remove the `rioboot` feature.

Reply:
I was actually wrong with the patch... since for me the test passes on a `nucleo-l496zg` so it must be something else...